### ISSUE TITLE
querying an inactive install throws unuseful error

### ISSUE THREAD

**Issue by margsuarez** [nycmeshnet/meshdb#650]
i clicked on an install number in the query list: https://db.mesh.nycmesh.net/admin/meshapi/install/9173
this was never installed (does not have NN and does not show "Active" in the "Install Status").  it launches MeshDB Admin anyway with the error "Install with ID “9173” doesn’t exist. Perhaps it was deleted?"

blank admin screen with one line of error is hard to parse visually.

perhaps in this case it should bring you to a different screen or display a popup.
and if it has an NN but is no longer Active, show appropriate message as well.

**Comment by Andrew-Dickinson** [comment]
Ah, this is just some broken links. Fixed here: https://github.com/nycmeshnet/meshforms/pull/107

**Comment by Andrew-Dickinson** [comment]
Whoops that didn't actually fix it, got it here: https://github.com/nycmeshnet/meshforms/pull/132
